releases/6.1.0.md

```diff
@@ -0,0 +1,101 @@
+Arches 6.1.0 release notes
+------------------------
+
+
+### Major enhancements
+
+### Additional highlights
+
+### Breaking changes
+
+### Dependency changes:
+```
+Python:
+    Upgraded:
+        elasticsearch 7.0.0 -> 7.11.0
+
+    Added:
+        none
+
+Javascript:
+    Upgraded:
+        none
+
+    Added:
+        none
+```
+
+
+### Upgrading Arches
+1. You must be upgraded to at least version 6.0 before proceeding.
+
+2. Be sure to backup your database before proceeding
+
+3. Upgrade to Arches 6.1.0
+
+        pip install --upgrade arches
+
+4. Within your project with your Python 3 virtual environment activated:
+
+        python manage.py migrate
+        python manage.py es reindex_database
+
+5. **Important:** If you are maintaining an Arches package, be sure to export your graphs and resource instance data before reimporting or reloading your package. 
+
+
+### Upgrading an Arches project
+
+1. Update your CACHES setting. In your project's settings.py file **add** the following CACHE to your CACHES setting. Be sure to do this in your settings_local.py file if that is where you have your CACHES setting:
```

Comment:
Yeah - new projects always overwrite the CACHES setting, so it will almost always be required unless for some reason they have deleted it from their project's settings.py file.

Reply:
ah -- good to know 👍